android-sdk/license_accepter.sh

```diff
@@ -20,9 +20,11 @@ accept_all_android_licenses() {
     echo "Android licenses directory doesn't exist, creating one..."
     mkdir -p $ANDROID_LICENSES
   fi
-  accept_license_of android-sdk-license 8933bad161af4178b1185d1a37fbf41ea5269c55
-  accept_license_of android-sdk-license d56f5187479451eabf01fb78af6dfcb131a6481e
+  accept_license_of android-googletv-license 601085b94cd77f0b54ff86406957099ebe79c4d6、
+  accept_license_of android-sdk-license 24333f8a63b6825ea9c5514f83c2829b004d1fee
   accept_license_of android-sdk-preview-license 84831b9409646a918e30573bab4c9c91346d8abd
+  accept_license_of android-sdk-preview-license 504667f4c0de7af1a06de9f4b1727b84351f2910
```

Comment:
For any new license added here, please add a corresponding raw license file [here](https://github.com/thyrlian/AndroidSDK/tree/master/EULA).  I have made an [example](https://gist.github.com/thyrlian/d1d0f264e47e32d790c785ac77749903) for you.

In case it's an updated license, please create a new license file, with date appended to the file name.  For instance: `AndroidSoftwareDevelopmentKitLicenseAgreement-20190116`.